chains/evm/calls/transactor/transact.go

```diff
@@ -0,0 +1,82 @@
+package transactor
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/rs/zerolog/log"
+)
+
+type TransactOptions struct {
+	GasLimit uint64
+	GasPrice *big.Int
+	Value    *big.Int
+}
+
+func NewDefaultTransactOptions() TransactOptions {
```

Comment:
I am thinking should I have this default constructor or should I just enable the user to pass `nill` as value for opts and then set these default values if `nill` is provided